app/templates/macros/toolkit_forms.html

```diff
@@ -1,9 +1,9 @@
-{% macro text(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
Yep this is just how Jinja does kwargs I'm afraid - you can't pass them in explicitly.  Pretty horrible design choice, I agree :/